Cargo.toml

```diff
@@ -1,89 +1,62 @@
-[package]
-name = "pevm"
-version = "0.1.0"
-edition = "2021"
-
-[features]
-optimism = [
-    "revm/optimism",
-    "dep:op-alloy-consensus",
-    "dep:op-alloy-network",
-    "dep:op-alloy-rpc-types",
+[workspace]
+default-members = ["bins/*"]
```

Comment:
This should be the core `pevm` library?

Reply:
Hmm, this just means that when we'll do `cargo build` it'll build binaries, but I can change it

Reply:
Yeah, the binary is very niche so most of time we lint, build, test, and bench the library by default (for development).